test/Dashboard.spec.js

```diff
@@ -70,4 +80,19 @@ describe('Dashboard tests', () => {
     expect(wrapper.find('.dash-sub-header').text()).toContain('(1)')
     expect(wrapper.find('.dash-container-info').exists()).toBe(true)
   })
+
+  test('Displays active products with My Asset Registries tile', async () => {
+    expect(wrapper.findComponent(dashboard).exists()).toBe(true)
+
+    // Add a delay because of products could be still loading
+    jest.advanceTimersByTime(250)
```

Comment:
The products are loading from where? Tests should not make actual API calls (those should be stubbed or mocked).

Timers in tests are always suspicious. If the code is simply async then you should be able to use nextTick() or flushPromises() ([example](https://github.com/bcgov/business-edit-ui/blob/223cc4f62e07396ab23ce73c98a65d2c2ee3d1dc/tests/unit/App.spec.ts#L355)).

Reply:
OK, I see it now.

How about updating your test comment to refer to this specific timeout on the dashboard?